cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_agent_config.json

```diff
@@ -620,5 +620,49 @@
       ],
       "feature_conditions": []
     }
-  ]
+  ],
+  "metric_configs": {
+    "metrics_collected": [
+      {
+        "metric_type": "mem",
+        "measurement": [
+          "mem_used_percent"
+        ],
+        "metrics_collection_interval": 60,
+        "append_dimensions": [
+          "ClusterName"
+        ],
+        "node_roles": [
+          "HeadNode"
+        ]
+      },
+      {
+        "metric_type": "disk",
+        "measurement": [
+          "disk_used_percent"
+        ],
+        "resources": [
+          "/"
+        ],
+        "metrics_collection_interval": 60,
+        "append_dimensions": [
+          "ClusterName"
+        ],
+        "node_roles": [
+          "HeadNode"
+        ]
+      }
+    ],
+    "append_dimensions": [
+      "InstanceId"
+    ],
+    "aggregation_dimensions": [
```

Comment:
Do we need aggregation_dimensions instance Id and cluster name here? Are they being used in the dashboard and alarm?

Reply:
Tried remove the aggregation dimensions. Memory metric was still fine but the graph for disk metric was broken. The reason is in Graphs and Alarms we use Instance Id as the dimension for these metrics. With aggregation dimension of Instance Id, both memory and disk metrics are fine. However, without it, disk metric has other dimensions beside Instance Id, such as device, file system, and path; memory metric doesn't have other dimensions so it was fine. The easiest way to fix it is to still use Instance Id as aggregation dimension. We don't need ClusterName so it can be safely removed.

Reply:
When using aggregation metric if InstanceId for disk metric, does it only include the value of “/” volume? Did you test if with adding "/" and "instance_id" without aggregation metrics in cloudwatch dashboard enough for monitoring disk usage for root volume?